modules/backend/patients/patient/treatments/Treatments.cpp

```diff
@@ -0,0 +1,11 @@
+#include "Treatments.hpp"
+
+namespace patient
+{
+
+void Treatments::updateTreatment([[maybe_unused]] const common::TreatmentState treatment_state)
+{
+    // move Treatment from planned to done
```

Comment:
We should mark it with [#128](https://github.com/coders-school/desktop-business-app/issues/128). As there should be more states and logic. 
Something like "state machine" should be created

Reply:
I will mark it as recommended.
For state machine I don't want to respond yet without #128 problem considerations.
It might be good for this case, but hard to tell without further stydying the problem.